cluster.c

```diff
@@ -1088,22 +1116,11 @@ void ShardingInfoReset(RedisRaftCtx *rr)
     si->shard_groups_num = 0;
 
     /* Reset array */
-    for (int i = 0; i < REDIS_RAFT_HASH_SLOTS; i++)
-        si->hash_slots_map[i] = NULL;
-
-    /* Add our local mapping */
-    ShardGroup sg = {
-        .slot_ranges_num = 0,
-        .slot_ranges = NULL,
-        .nodes_num = 0,
-        .nodes = NULL
-    };
-
-    FillShardSlots(rr, &sg);
-
-    RRStatus ret = ShardingInfoAddShardGroup(rr, &sg);
-    RedisModule_Assert(ret == RR_OK);
-    ShardGroupTerm(&sg);
```

Comment:
this work is now done by AddBasicLocalShardGroup() if sharding info doesn't exist otherwise at the appropriate point in time (i.e. init / join / reload)